### PR TITLE
Update gpio_example_main.c (IDFGH-9357)

### PR DESCRIPTION
**Pull request by rubeng0nzalez**
Add comments to clarify how the bits of GPIO_OUTPUT_PIN_SEL and GPIO_INPUT_PIN_SEL are defined after the OR bit operator

### CODE REVIEW

File: examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c [hunk 1]
```diff
@@ -38,9 +38,21 @@
 #define GPIO_OUTPUT_IO_0    CONFIG_GPIO_OUTPUT_0
 #define GPIO_OUTPUT_IO_1    CONFIG_GPIO_OUTPUT_1
 #define GPIO_OUTPUT_PIN_SEL  ((1ULL<<GPIO_OUTPUT_IO_0) | (1ULL<<GPIO_OUTPUT_IO_1))
+/*
+ * In binary representation,
+ * 1ULL<<GPIO_OUTPUT_IO_0 is equal to 0000000000000000000001000000000000000000 and
+ * 1ULL<<GPIO_OUTPUT_IO_1 is equal to 0000000000000000000010000000000000000000
+ * GPIO_OUTPUT_PIN_SEL                0000000000000000000011000000000000000000
+ * */
```
Comment (suda-morris) [inline examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c:46]: I'm afraid this comment can be confusing if user changed the GPIO number in the menuconfig. Please add an assumption statement to the note, e.g. "Let's say, GPIO_OUTPUT_IO_0=18, GPIO_OUTPUT_IO_1=19".

**Review by suda-morris — CHANGES_REQUESTED** [review]

File: examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c (hunk 1 above)
Comment (rubeng0nzalez) [reply examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c:46]: Hi @suda-morris , I agree. Will make the change and see if this woks better.

Thanks

Comment (rubeng0nzalez) [reply examples/peripherals/gpio/generic_gpio/main/gpio_example_main.c:47]: Hi @suda-morris , I just made the change. What do you think?👍

**Comment by suda-morris** [comment]
sha=9390b9247f2882d1a30c34c3163c0fa121075253

**Pull request closed without merging**